### ISSUE TITLE
Ge_patch conflicts with cheat plugins like cwcheat

### ISSUE THREAD

**Issue by DiegooBedin** [TheOfficialFloW/GePatch#19]
Ge_patch conflicts with cheat plugins like cwcheat. For example, in games like PSP's Need for Speed Most Wanted (USA), gepatch prevents cwcheat from showing the menu to select cheats

**Comment by Victjor** [comment]
Yeah, this is a bit unfortunate, since with Ge_patch you can enable cheats and patches that increase FPS and what not, which would be amazing with the GE_Patch, I really hope there's a solution to this if possible.
